.github/workflows/integration-tests.yml

```
@@ -0,0 +1,392 @@
+          # Test credentials
+          TEST_ADMIN_PASSWORD=test-password-123
+          ADMIN_SECRET=test-admin-secret-key-minimum-32-characters
+          EOF
+
+      - name: "🚀 Start Development Server"
+        run: |
+          echo "Starting development server for E2E testing..."
+          npm run dev &
+          SERVER_PID=$!
+          echo "SERVER_PID=$SERVER_PID" >> $GITHUB_ENV
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)